### ISSUE TITLE
v2: Default Organization and bucket set in config

### ISSUE THREAD

**Issue by leifnel** [mblackstock/node-red-contrib-influxdb#91]
It would be nice if the configuration for v2 also included organizsation and bucket, instead of having to configure it on each in/out node.

In the rare case of having to use multiple organization/buckets, use a configuration for each.

**Comment by mblackstock** [comment]
I agree, in fact, this was the original model for configuration.  It looks like we introduced this when we added 1.8 and 2.0 support.  I'm thinking to remove the config on the in/out/batch node even if this may break some flows.

**Comment by mirkocomparetti-synesis** [comment]
I would consider to have the bucket in the msg structure as well, just like the measurement.

**Comment by ycsin** [comment]
+1 on this, so that it can be more "programmable" instead of having to create one node for every org/bucket

**Comment by greg-mcnamara** [comment]
+1 from me too. I already have organization in the incoming message topic, so it would be great to route this through a single node.

**Comment by offshore42** [comment]
+1, it would make it more versatile
